PWGJE/Tasks/jetfinderQA.cxx

```diff
@@ -623,19 +624,33 @@ struct JetFinderQATask {
   }
   PROCESS_SWITCH(JetFinderQATask, processJetsMCDWeighted, "jet finder QA mcd with weighted events", false);
 
-  void processJetsMCP(soa::Join<aod::ChargedMCParticleLevelJets, aod::ChargedMCParticleLevelJetConstituents>::iterator const& jet, JetParticles const&)
+  PresliceUnsorted<soa::Filtered<JetCollisionsMCD>> CollisionsPerMcpJets = aod::jmccollisionlb::mcCollisionId;
```

Comment:
The name should be changed from CollisionsPerMcpJets to CollisionsPerMCPCollision